entries/blur.xml

```diff
@@ -1,66 +1,108 @@
+      <pre><code>
+  $( "#other" ).on( "click", function() {
+    $( "#target" ).trigger( "blur" );
+  } );
+      </code></pre>
+      <p>After this code executes, clicks on <samp>Trigger the handler</samp> will also alert the message.</p>
+      <p>The <code>blur</code> event does not bubble in Internet Explorer. Therefore, scripts that rely on event delegation with the <code>blur</code> event will not work consistently across browsers. As of version 1.4.2, however, jQuery works around this limitation by mapping <code>blur</code> to the <code>focusout</code> event in its event delegation methods, <a href="/live/"><code>.live()</code></a> and <a href="/delegate/"><code>.delegate()</code></a>.</p>
+    </longdesc>
+    <note id="detach-shorthand" type="additional" data-event="blur"/>
+    <example>
+      <desc>To trigger the blur event on all paragraphs:</desc>
+      <code><![CDATA[
+  $( "p" ).trigger( "blur" );
+  ]]></code>
+    </example>
+    <category slug="events/form-events"/>
+    <category slug="forms"/>
+    <category slug="version/1.7"/>
+  </entry>
+
+  <entry type="method" name="trigger" return="jQuery">
+    <title>.trigger( "blur" )</title>
+    <signature>
+      <added>1.7</added>
+      <argument name="&quot;blur&quot;" type="string">
+        <desc>The string <code>"blur"</code>.</desc>
+      </argument>
+    </signature>
+    <longdesc>
+      <p>See the description for <a href="#on1"><code>.on( "blur", ... )</code></a>.</p>
+    </longdesc>
+    <category slug="events/form-events"/>
+    <category slug="forms"/>
+    <category slug="version/1.7"/>
+  </entry>
+
+  <entry type="method" name="blur" return="jQuery">
+    <title>.blur()</title>
+    <signature>
+      <added>1.0</added>
+      <argument name="handler" type="Function">
+        <desc>A function to execute each time the event is triggered.</desc>
+        <argument name="eventObject" type="Event" />
+      </argument>
+    </signature>
+    <signature>
+      <added>1.4.3</added>
+      <argument name="eventData" type="Anything" optional="true">
+        <desc>An object containing data that will be passed to the event handler.</desc>
+      </argument>
+      <argument name="handler" type="Function">
+        <desc>A function to execute each time the event is triggered.</desc>
+        <argument name="eventObject" type="Event" />
+      </argument>
+    </signature>
+    <signature>
+      <added>1.0</added>
+    </signature>
+    <longdesc>
+      <p><code>.blur( handler )</code> &amp; <code>.blur( eventData, handler )</code> are deprecated aliases of <code>.on( "blur", handler )</code> &amp; <code>.on( "blur", eventData, handler )</code> respectively.</p>
+      <p><code>.blur()</code> is a deprecated alias of <code>.trigger( "blur" )</code>.</p>
+    </longdesc>
+    <category slug="events/form-events"/>
+    <category slug="forms"/>
+    <category slug="version/1.0"/>
+    <category slug="version/1.4.3"/>
+    <category slug="deprecated/deprecated-3.3"/>
```

Comment:
Note to self: this is not visible in the UI, I need to add proper metadata to each sub-entry.